### ISSUE TITLE
Add support for column compression for PS 5.6/5.7

### ISSUE THREAD

**Issue by ShahriyarR** [Percona-QA/PTB#13]
After release, there is need to add column compression support into PTB.

**Comment by ShahriyarR** [comment]
Currently supported only on PS 5.6.33

**Comment by RoelVdP** [comment]
Yes. 5.7 is underway. Perhaps we can leave this open for little bit and then add it.

**Comment by ShahriyarR** [comment]
NOTE:
Main differences can be affected with 5.7, is "General Tablespace" and "Generated Columns".
It will be interesting to test them with "Column Compression" + sysbench run.
